indra/llwindow/llwindowwin32.cpp

```diff
@@ -970,6 +972,11 @@ void LLWindowWin32::close()
 
     LL_DEBUGS("Window") << "Destroying Window" << LL_ENDL;
 
+    if (sWindowHandleForMessageBox == mWindowHandle)
+    {
+        sWindowHandleForMessageBox = NULL;
+    }
+
```

Comment:
If window somehow gets closed without without a call for close (was the norm untill recently), sWindowHandleForMessageBox will stay invalid. I suggest checking it again in ~LLWindowWin32()

Reply:
Done